opencti-platform/opencti-front/src/schema/relay.schema.graphql

```diff
@@ -10163,7 +10163,7 @@ input MalwareAnalysisAddInput {
   result_name: String!
   result: String
   analysisSco: [String]
-  sample: String
+  analysisSample: String
```

Comment:
This is the only breaking change to the API.

All other renaming are on attribute never exposed in the API for now.